frontend/src/main/scala/akkaviz/frontend/components/Alert.scala

```diff
@@ -19,31 +19,26 @@ class Alert extends Component {
     parent.appendChild(connectionAlert)
   }
 
-  def success(msg: String, fn: MouseEvent => Unit = _ => {}) = {
+  private[this] def makeAlert(msg: String, currentFlag: String, fn: MouseEvent => Unit = _ => {}): Unit = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should not name methods after their defining object](https://www.codacy.com/app/lustefaniak/akka-viz/file/2204417616/issues/source?bid=3121282&fileBranchId=3170129#l22)